lib/rosette/queuing/commits.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: UTF-8
+
+module Rosette
+  module Queuing
+    module Commits
+
+      autoload :CommitConductor, 'rosette/queuing/commits/commit_conductor'
```

Comment:
if activesupport is a dependency, you can probably skip this by adding to the autoload paths